core/azure-core/src/main/java/com/azure/core/util/polling/Poller.java

```diff
@@ -226,6 +226,47 @@ public void cancelOperation() throws UnsupportedOperationException {
         return this.fluxHandle.blockLast();
     }
 
+    /**
+     * Blocks until given {@link OperationStatus} is received or a timeout expires if provided. A {@code null} {@code timeout} will cause to block indefinitely for desired status.
+     * @param statusToBlockFor The {@link OperationStatus} to block for and it can be any valid {@link OperationStatus} value.
+     * @param timeout The time after which it will stop blocking. A {@code null} value will cause to block indefinitely. Zero or negative are not valid values.
+     * @return {@link PollResponse} matching the status to block for or times out.
+     * @throws IllegalArgumentException if {@code timeout} is zero or negative. If {@code statusToBlockFor} is {@code null}.
+     */
+    public PollResponse<T> blockUntil(OperationStatus statusToBlockFor, Duration timeout) {
```

Comment:
Did you remove the blockUntil(OperationStatus) overload? I think it should still remain.

Reply:
I see it on my local, looks like I did not check in my changes. I will redo.

Reply:
I still don't see the method?

Reply:
added this method back in.

Reply:
Added  the API back in